py3status/modules/do_not_disturb.py

```diff
@@ -135,7 +150,7 @@ class Py3status:
     thresholds = [(0, "bad"), (1, "good")]
 
     def post_config_hook(self):
-        servers = ["xfce4-notifyd", "dunst", None]
+        servers = ["xfce4-notifyd", "dunst", "mako", None]
```

Comment:
If you have `mako` and `dunst` installed, but are on `sway`, this would pick `dunst`. I wonder if mako should be checked first before dunst. `xfce4-notifyd` (or any other non-i3/sway notification servers) should always be last anyway.

(Thought) I wonder if `mako` would work on `i3`. If not, I could imagine some users switching between i3/sway sometimes to test py3status or other things... resulting in wrong server (i.e  `mako` on i3). I wonder if we need to check for `sway` too to see if we ought to run `mako` or `dunst`. Maybe it's an overkill. I don't know.

Reply:
`mako` doesn't work on i3, it was deliberate decision to prefer i3, as it is quite common for people to have both working i3 and sway in experimental mode, but once people switch to sway they no longer keep i3. I think it's fine like this (except your point that I agree with, we should move `mako` before `xfce4-notifyd`), defaults are defaults but users can control what they want in the config.

Reply:
It is trivial to check for `sway` on `post_config_hook` to see if we need to `servers.insert(1, "mako")` before proceeding. Otherwise, this can auto pick mako on i3 if it is installed. 

>defaults are defaults but ...

Defaults are useful if they are smart enough to know which servers are supported.

>... but users can control what they want in the config.

Fine. As long as you acknowledged this can happen... Telling all users to change / control / fix their config instead of us auto detecting things like this in `post_config_hook` seems meh.

Carry on regardless. :+1: